create3_lidar/README.md

```diff
@@ -0,0 +1,98 @@
+# iRobot® Create® 3 LIDAR SLAM demo
+
+This example sets up LIDAR SLAM with a Create® 3 robot and Slamtec RPLIDAR spinning laser rangefinder.
+
+## Parts List
+
+* Raspberry Pi® 4[^1]
+* USB-C® to USB-C® cable[^2]
+* Slamtec RPLidar A1M8
+* USB Micro B to USB A cable
+* 4x M2.5 x 8 flat head machine screw
+* 4x M2.5 x 6 self-tapping screw
+* 10x M3 x 10 self-tapping screw
+
+## Setup
+
+### Hardware Setup
+
+The files in this example assume the RPLIDAR is mounted 12 mm behind the center of rotation on the top of the Create 3 robot, in the arrangement shown below.
+The SLAM solver relies on a proper `tf` tree; if you wish to mount the sensor in another location, you will need to modify the parameters in the static transform publisher launched from `launch/sensors_launch.py`.
+
+Note that all STLs of brackets referenced in this example are found in our [create3_docs](https://github.com/iRobotEducation/create3_docs/tree/main/docs/hw/data/brackets) repository.
+
+![Image of Create 3 showing setup and placement of sensors](https://iroboteducation.github.io/create3_docs/examples/data/create3_lidar_top.jpg)
+
+1. Affix a LIDAR to your robot.
+   STLs are available to mount an RPLidar A1M8 as well as its USB adapter.
+   The LIDAR can be attached to the mounting plate using qty. 4 of M2.5 x 8 screws.
+   The USB adapter can be attached to its mounting plate by heat staking (we used a soldering iron with an old tip in a well-ventilated space).
+   Both mounting plates can be attached to the Create® 3 faceplate with M3 x 10 self-tapping screws.
+
+1. Affix an SBC to your robot.
+   In this example, we use a Raspberry Pi® 4, but other devices should work.
+   STLs are available to mount a Raspberry Pi® to the Create® 3 robot.
+   The Raspberry Pi® can be attached to the mounting plate using qty. 4 of M2.5 x 6 self-tapping screws.
+   The mounting plate can be attached to the Create® 3 cargo bay with M3 x 10 self-tapping screws.
+
+1. Connect the LIDAR to the SBC with a USB Micro B to USB A cable.
+   Thread the cable through the passthrough in the top of the robot.
+
+1. Connect the SBC to the Create® 3 adapter board with a USB-C® to USB-C® cable.
+
+### SBC Setup
+
+On the SBC, clone and build this, and source the setup shell scripts.
+
+Start the sensors launch script, which includes the LIDAR driver and transform from the laser scan to the robot:
```

Comment:
I suggest to add a third launch file that includes
 - sensor_launch.py
 - slam_toolbox_launch.py
 - starts rviz using our config

Reply:
oh right, nevertheless we can simplify the process by having 2 launch files:
 - rviz
 - sensors + slam

Reply:
@justinIRBT also suggested this; I was going to leave it as is to try to separate the functionality, but to your point if this is just a LIDAR SLAM example, there's no reason to do that.